### DISCUSSION TITLE
Create codespaces with xvfb

### DISCUSSION THREAD

**Discussion by melissawm** [community/community#55090] [Codespaces]
### Select Topic Area

Question

### Body

Is it possible to create codespaces with [xvfb](https://www.x.org/releases/X11R7.6/doc/man/man1/Xvfb.1.xhtml) installed/enabled, so we can run headless GUI applications? See, for example, https://github.com/aganders3/headless-gui

This would be useful for testing, for example.

**Comment by craiglpeters** [comment]
Yay! No blockers to using this in Codespaces. In fact we produced a feature [desktop-lite](https://www.google.com/url?q=https://github.com/devcontainers/features/tree/main/src/desktop-lite] that implements a full X11 desktop. Would this help?

**Comment by melissawm** [comment]
This looks great, for some reason I didn't find it in my previous search - thank you!
